scipy/stats/_stats_py.py

```diff
@@ -3365,6 +3368,10 @@ def gstd(a, axis=0, ddof=1):
                       'normal': special.erfinv(0.5) * 2.0 * math.sqrt(2.0)}
 
 
+@_axis_nan_policy_factory(
+    lambda x: x, result_to_tuple=lambda x: (x,), n_outputs=1,
+    default_axis=None, override={'nan_propagation': False}
```

Comment:
I'm curious - you (discovered and) decided to use `override={'nan_propagation': False}`. I can imagine that the function does something special with NaNs because it uses quartiles - what does it do that is different?

Reply:
I had to do this because otherwise it would not run the underlying function at all if you have some `nan`, hence not validate the inputs. So using `scale='foobar'` would not raise.